### PR TITLE
Feature/List Spaces 

### LINKED ISSUE #19: As a proprietor, I want to be able to see all of my spaces listed on a page. (epic-account-page)

### PR DESCRIPTION
**Pull request by bishalr1**
This is for story #19 
This basically took the spaces that are assigned or created by each proprietor and made them show up on the account page for the specific proprietor. This was mostly already done by Binh. But we were missing a few things to make it fully functional. As we were missing a button that connects it to the times. I added that and also made a page for the said time and dates to show up in an orderly manner. It is shown in the order of the primary key. That will be fixed later on.

Testing:
one main test is done for this and it is to make sure users and spaces are connected.
That test is in test.py: ListSpacesTest

### CODE REVIEW

File: sadcowpng/sharedspaces/templates/sharedspaces/date_time.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                            </h5>
+                            <a href="{% url 'update_date_time' date_time.id %}" class="btn btn-primary">Deactivate</a>
+                        </div>
+                    </div>
+                {% endfor %}
+            </div>
+            {% endif %}
+        </div>
+    </body>
+</html>
```
Comment (bishalr1) [inline sadcowpng/sharedspaces/templates/sharedspaces/date_time.html:44]: Will fix this when I do space reuse.

File: sadcowpng/sharedspaces/views.py [hunk 2]
```diff
@@ -258,3 +258,23 @@ def update_space_date_time(request, data_time_id):
                    "id": data_time_id}
 
     return render(request, 'sharedspaces/update_space_date_time.html', context=context)
+
+
+@login_required
```
Comment (btruong9) [inline sadcowpng/sharedspaces/views.py:263]: You could also add a user_is_entry_owner decorator after login_required too.

**Review by btruong9 — APPROVED** [review]
lgtm!

File: sadcowpng/sharedspaces/views.py (hunk 2 above)
Comment (bishalr1) [reply sadcowpng/sharedspaces/views.py:263]: Fixed it in feature/space-reuse

**Pull request merged**